home/models/account.py

```diff
@@ -13,6 +19,12 @@ class Account(models.Model):
     name = models.CharField(max_length=250, help_text="User's name")
     zip = models.CharField(max_length=25, help_text="User's zipcode")
     age = models.IntegerField(help_text="User's age")
+    is_sf_resident = models.BooleanField(null=True, help_text="Whether the user is a SF resident or not, based on zip")
+    is_latino = models.BooleanField(null=True, blank=True, help_text="Latino or Hispanic origin")
+    race = models.ManyToManyField("Race", blank=True, help_text="Self-identified race(s) of user")
+    gender = models.CharField(max_length=25, blank=True, help_text="Self-identified gender identity of user")
```

Comment:
Should you also have a table for gender identities similar to race? Some sample guidelines for gender and race are at: 

https://sfmohcd.org/sites/default/files/Documents/Demographics/MOHCD%20Guidelines%20for%20Collecting%20Sex%20and%20Gender%20and%20Sexual%20Orientation%20Data%206-2020.pdf

https://sfmohcd.org/sites/default/files/Documents/Demographics/Revised%20Guidelines%20for%20Categorizing%20Race%20and%20Ethnicity%20Feb%202020.pdf